frontend/src/ContentArea/ContentArea.js

```diff
@@ -1,43 +1,104 @@
 import React from 'react';
-import JSONToHTML from '../JSONHandle';
+import { Redirect } from 'react-router';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import marked from 'marked';
 
 export default class ContentArea extends React.Component {
-  constructor(props) {
-    super(props);
+  componentWillMount() {}
 
-    this.state = {
-      innerHTML: undefined,
-    };
+  render() {
+    const { path } = this.props;
+    let view = 'none';
+    const id = JSON.parse(localStorage.getItem('pathTable'))[path] || '';
+    const articles = JSON.parse(localStorage.getItem('allArticles'));
+    if (id) {
+      // exists id for that path
+      const index = JSON.parse(localStorage.getItem('articleIndex'));
+      const options = { __html: marked(articles[index[id]].content) };
```

Comment:
getting picky now ... isn't this the html version of the article content? Why is the var called options?

Reply:
yes it is html code, but in view it displays raw html code with all tags on page.  To get around i use dangerouslySetInnerHtml in next line :  `view = <div dangerouslySetInnerHTML={options} />;`
dangerouslySetInnerHtml receives object with schema: `{__html : <html here>}`.    This object i call 'options'.   I think it is not critical so if you have another name for object, i listen. Anyway, we can call it MichaelSchumacher or moonWalker123 but it still would be object which we pass to dangerouslySetInerHtml.

Reply:
it's fine I was only referring to the name. I didn't realise it was called options in the dox. Seems a bit odd. `options` will do fine